cpp/iedriver/BrowserFactory.cpp

```diff
@@ -1393,4 +1435,50 @@ bool BrowserFactory::IsEdgeMode() const {
   return this->edge_ie_mode_;
 }
 
+// delete a folder recursively
+int DeleteDirectory(const std::wstring &dir_name) {
+  WIN32_FIND_DATA file_info;      
+
+  std::wstring file_pattern = dir_name + L"\\*.*";
+  HANDLE hFile = ::FindFirstFileW(file_pattern.c_str(), &file_info);
+  if(hFile != INVALID_HANDLE_VALUE) {
+    do {
+      if(file_info.cFileName[0] == '.') continue;
+      std::wstring file_path = dir_name + L"\\" + file_info.cFileName;
+
+      if(file_info.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) {
+        int iRC = DeleteDirectory(file_path);
```

Comment:
Prefer `return_value` to `iRC`

Reply:
Updated variable name to `return_value`.